docs/index.md

```diff
@@ -0,0 +1 @@
+# JupyterLab Plugin Playground
```

Comment:
Perhaps:

    > [Try `jupyterlab-plugin-playground` in your browser right now ![try-badge]][try]

    [try]: ./lite/lab/index.html?path=example.js
    [try-badge]: https://jupyterlite.rtfd.io/en/latest/_static/badge.svg


    ```{include} ../README.md
    ```